keyboards/boardsource/holiday/spooky/keymaps/rip_mx/keymap.c

```diff
@@ -0,0 +1,78 @@
+/* Copyright 2020 boardsource
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [_BASE] = LAYOUT(
+        RGB_TOG, KC__VOLUP,  KC_F2,
+        RGB_MOD, KC__VOLDOWN,KC_F1
```

Comment:
The default volume keycodes should work on Windows, MacOS and Linux. No need to use the custom ones for mac. 
```suggestion
        RGB_TOG, KC_VOLU, KC_F2,
        RGB_MOD, KC_VOLD, KC_F1
```

Reply:
my bad thank you